strange_function.py

```diff
@@ -0,0 +1,34 @@
+# HAcker Rank Problem: https://www.hackerrank.com/challenges/the-strange-function/problem
+#!/bin/python3
+
+import sys
+
+def find_gcd(x, y): 
+    while(y):

```

Comment:
A comment is important for this as you're implementing a well-known algorithm. It would also give the reader a lot of information about your implementation (and whether it's correct).